doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -27,86 +26,54 @@ a suitable set of meteorological variables can be used to distinguish different
-across a variety of weather types through the use of a manually constructed decision tree model
-taking an appropriate set of feature parameters (meteorological variables) as inputs.
-Details of this method can be found in `Hewson & Pillosu, 2021`_. 
-
-One advantage of the ECPoint approach is that the calibration is inherently non-local. As
-the calibration is done by identifying distinct weather types, the model bias and scale
-difference should be independent of any given location and time as the underlying physical
-process should be identical. Thus a grid-point can be calibrated using data from any location,
-provided the underlying weather type is the consistent. This enables effective calibration to
-be applied to areas that are typically lacking sufficient cases to calibrate against.
+Rainforests (like ECPoint) processes each ensemble member separately to produce a per-realization output that can be
+considered a pseudo-ensemble representing the
+conditional probability distribution that describes the likelihood of observing a given outcome when the
+realised atmospheric state is consistent with that represented in the input ensemble member forecast.
+The predicted distributions for different ensemble members are then blended in probability space to produce 
+the final calibrated probability output.
+
+One advantage of the ECPoint approach is that the calibration is inherently non-local. As the calibration is done by
+identifying distinct weather types, the model bias and scale difference should be independent of any given location and time as 
+the underlying physical process should be identical. Thus a grid-point can be calibrated using data from any location, provided the 
+underlying weather type is the consistent. This enables effective calibration to be applied to areas that are typically lacking 
+sufficient cases to calibrate against.
+
 
 ****************************
 The RainForests method
 ****************************
 
-RainForests is an adaptation of the ECPoint method that seeks to use machine learning based
+RainForests is an adaptation of the ECPoint method that uses machine learning based
 tree models to replace the manually constructed tree model of ECPoint. Here we use gradient-boosted
-decision tree (GBDT) ensemble models.
-
-The underlying principle of the calibration methodology is essentially the same, namely using
-a set of feature variables to map onto error distributions which can be applied to the
-forecast values to produce calibrated forecast values. However, the way in which these
-distributions are constructed differs somewhat within the RainForests framework.
-
-Our approach is to use a series of GBDT models, taking the feature variables as inputs, to
-produce exceedance probability values for representative error thresholds. Collectively
-these represent the error distributions as a cumulative distribution function.
+decision tree (GBDT) models.
```

Comment:
```suggestion
```

Reply:
what did you mean here? it seems the suggested change just deleted the end of the sentence...

Reply:
Sorry, I had another suggestion above this that hasn't seemed to have made it in. I'll add that comment in too... given this straddles both additions and removals, I was trying to frame the suggestions so that they could easily slot in (but I obviously failed).